macosx/ProgressBarView.mm

```diff
@@ -183,7 +191,14 @@ - (void)drawPiecesBar:(NSRect)barRect forTorrent:(Torrent*)torrent
         }
         else
         {
-            pieceColor = [NSColor.whiteColor blendedColorWithFraction:piecesPercent[i] ofColor:self.fBluePieceColor];
+            if (NSApp.darkMode)
```

Comment:
Since this is in a loop and condition result doesn't (shouldn't) change, please move out, e.g.
```objc++
NSColor* const pieceBgColor = NSApp.darkMode ? NSColor.controlColor : NSColor.whiteColor;

for (int i = 0; i < pieceCount; i++)
{
    ...
    pieceColor = [pieceBgColor blendedColorWithFraction:piecesPercent[i] ofColor:self.fBluePieceColor];
    ...
}
```

Reply:
@nevack can you address it in order to get the fix merged?

Reply:
Yes, will do this soon with a rebase.
Thanks for bringing this up!

Reply:
Done